exercises/concept/the-farm/.docs/instructions.md

```diff
@@ -12,33 +12,16 @@ While this is good for your pocket, you want to catch the division by zero retur
 Also, your silly nephew (who has just learned about negative numbers) sometimes will say that there are a negative number of cows.
 You love your nephew so you want to return a helpful error when he does that.
 
-## 1. Get the amount of fodder from the `weightFodder` function
+## 1. Get the amount of fodder from the `FodderAmount` method
 
-You will be passed a `WeightFodder` which has a function called `FodderAmount` which returns the amount of fodder available and possibly an error.
-
-```go
-// twentyFodderNoError says there are 20.0 fodder
-fodder, err := DivideFood(twentyFodderNoError, 10)
-// fodder == 2.0
-// err == nil
-```
-
-If `ErrScaleMalfunction` is returned by `FodderAmount`, double the fodder amount returned by `FodderAmount`.
-For any other error, return `0` and the error.
-
-```go
-// twentyFodderNoError says there are 20.0 fodder and a ErrScaleMalfunction
-fodder, err := DivideFood(twentyFodderWithErrScaleMalfunction, 10)
-// fodder == 4.0
-// err == nil
-```
+You will be passed a `WeightFodder` interface instance which has a method called `FodderAmount` that returns the amount of fodder available and possibly an error.
```

Comment:
In your new version, task 1 does not contain any actual task anymore. There is only this one sentence. That is confusing.

Reply:
Agreed. With the new set of changes, this is no longer needed and hence removed.